src/riak_cs_app.erl

```diff
@@ -57,14 +55,11 @@ stop(_State) ->
 sanity_check(true, {ok, true}) ->
     riak_cs_sup:start_link();
 sanity_check(false, _) ->
-    _ = lager:error("You must update your Riak CS app.config. Please see the"
+    _ = lager:error("You must update your Riak CS riak-cs.conf. Please see the"
                     "release notes for more information on updating you"
```

Comment:
It's better to add white spaces at the end of these two lines because concatenated string is shown in the log.